scripts/components/leaderboard_entry.js

```diff
@@ -64,6 +64,17 @@ class LeaderboardEntry {
 			});
 		}
 
+		if (timeData.type === LEADERBOARD_ENTRY_TYPE.ONLINE || timeData.type === LEADERBOARD_ENTRY_TYPE.ONLINE_CACHED) {
+			items.push({
+				label: 'View on website',
+				icon: 'file://{images}/online/publiclobby.svg',
+				style: 'icon-color-blue',
+				jsCallback: () => {
+					SteamOverlayAPI.OpenURL(`https://momentum-mod.org/dashboard/runs/${timeData.replayID}`)
```

Comment:
Would be better to use `OpenURLModal` here:

```suggestion
					SteamOverlayAPI.OpenURLModal(`https://momentum-mod.org/dashboard/runs/${timeData.replayID}`)
```

Allows users to quickly get back in game if they didn't mean to click it.